manifests/config.pp

```diff
@@ -83,15 +83,15 @@
   if $config_manage {
     if $source_config_file {
       file { $config_file:
-        ensure  => file,
```

Comment:
why change this to present, file is much more descriptive and does the same thing (i thought puppet-lint flagged this but dont see it on their we site)